README.md

```diff
@@ -11,7 +11,7 @@ Mina is saves API server responses to disk and serves them with its own HTTP ser
 
 ## Install
 
-    go get -u github.com/sariina/mina
+`go get -u github.com/sariina/mina`
```

Comment:
As I said , I made changes with my cellphone and I couldn't use indents :cry:
Honestly , I don't know why indents not working ...